modules/update-process-workflow.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/how-updates-work.adoc
+
+:_content-type: CONCEPT
+[id="update-process-workflow_{context}"]
+= Update process workflow
+
+The following represents a detailed workflow of the {product-title} (OCP) update process.
+
+. The target version is stored in the `spec.desiredUpdate.version` field of the ClusterVersion resource, which may be managed through the web console or the CLI.
```

Comment:
Throughout:
s/ClusterVersion resource/`ClusterVersion` resource 

Using markup seems to be the common way to go